sqlite-android/src/main/java/io/requery/android/database/CursorWindow.java

```diff
@@ -476,4 +500,8 @@ protected void onAllReferencesReleased() {
     public String toString() {
         return getName() + " {" + Long.toHexString(mWindowPtr) + "}";
     }
+
+    public static int getWindowSizeBytes() {
```

Comment:
Need to remove static from here

Reply:
Done. Sorry, it used to make sens when it was constant and did not rethink my PR.